tests/testthat/test-run.R

```diff
@@ -987,3 +987,115 @@ test_that("parameters passed to dependency resolution include defaults", {
     path_orderly_run_rds(file.path(root, "draft", "use_dependency", id)))
   expect_equal(info$meta$depends$id, dat$ids[[1]])
 })
+
+test_that("failed run creates failed rds", {
+  testthat::skip_on_cran()
```

Comment:
Not sure, just copying and pasting from other places. I think every other example where the dir is setup via `prepare_orderly_git_example` is skipped on CRAN so I copied that over too.

Reply:
Oh this creates a dir `minimal` in the current working directory. That's because we are passing `minimal` to `prepare_orderly_git_example` instead of using a tempdir. Will tidy that up here.